internal/backends/winit/glwindow.rs

```diff
@@ -418,6 +418,28 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
                         winit_window.set_inner_size(new_size);
                     }
                 }
+
+                // Auto-resize to the preferred size if users (SlintPad) requests it
+                #[cfg(target_arch = "wasm32")]
+                {
+                    let canvas = self.renderer.html_canvas_element();
+
+                    if canvas
+                        .dataset()
+                        .get("slint-auto-resize-to-preferred")
```

Comment:
who does the mapping between camel case and kebab case?

Reply:
The browser does it (yes, that's quite magic): https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset